src/content/ja/2020/accessibility.md

```diff
@@ -0,0 +1,583 @@
+
+悲しいことに、私たちや[WebAIM Million](https://webaim.org/projects/million/)のような分析を行っている他のチームは、年々これらの指標にほとんど改善が見られず場合によっては全く改善が見られないことを発見しています。すべての[Lighthouse Accessibility](https://web.dev/lighthouse-accessibility/)監査データのサイト全体のスコアの中央値は、2019年の73%から2020年には80%に上昇しました。この7%の上昇は、正しい方向へのシフトを表していると期待しています。しかし、これらは自動化されたチェックであり、開発者がルールエンジンを裏切る良い仕事をしていることを意味する可能性があるので慎重に楽観視しています。
+
+当社の分析は、自動化されたメトリクスのみに基づいています。自動化されたテストは、インターフェイスに存在しうるアクセシビリティの障壁のほんの一部を捉えているに過ぎないことを覚えておくことが重要です。アクセシブルなサイトやアプリケーションを実現するためには、手動テストや障害者によるユーザビリティ・テストを含む定性分析が必要です。
+
+私たちの最も興味深い洞察を5つのカテゴリーに分けました。
+
+1. 読みやすさ。
+2. ウェブ上のメディア。
+3. ページナビゲーションのしやすさ。
+4. ウェブ上の支援技術。
+5. フォームコントロールのアクセシビリティ。
+
+私たちはウェブ上での身の引き締まるようなメトリクスと明らかなアクセシビリティの怠慢が満載のこの章が、読者にこの仕事の優先順位をつけ、より包括的で公正なインターネットへと移行していくための習慣を変えるきっかけとなることを願っています。
+
+## 読みやすさ
+
+コンテンツをできるだけシンプルで読みやすいものにすることは、ウェブアクセシビリティの重要な側面です。ページのコンテンツを読むことができないと、ユーザーはウェブサイトでのタスクを完了できません。ウェブページには色のコントラスト、ページのズームや拡大縮小、言語の識別など、読みやすくしたり、読みにくくしたりする多くの側面があります。
+
+### カラーコントラスト
+
+ページのコントラストが高いほど、テキストベースのコンテンツを見やすくなります。コントラストの低いコンテンツを表示することが困難な人には色覚障害のある人、軽度から中等度の視力低下のある人、明るい光の中での画面のまぶしさなど、文脈上コンテンツを表示することが困難な人を含みます。
+
+{{ figure_markup(
+  image="sites-with-sufficient-color-contrast-2019-2020.png",
+  caption="色のコントラストが十分なサイト。",
+  description="2019年と2020年に十分なカラーコントラストを持つサイトの割合を示す棒グラフ。2019年に十分なカラーコントラストを持っていたサイトの22.04%が2020年には21.06%に減少し、77.96%のサイトが2019年にはカラーコントラストが不十分であったことを意味し、2020年には78.94%に増加しています。",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vSjkB_XAYiwkYrMuoXp44mdqMGJVDUkTr_48ELozY72Mdv3OlxeWV9ysbDY9bs6hA7LnJTrHar9aZlM/pubchart?oid=1827221015&format=interactive",
+  sheets_gid="1115686547",
+  sql_file="color_contrast.sql"
+  )
+}}
+
+残念ながら、十分なカラーコントラストを持つサイトは21.06%しかありませんでした。これは昨年のすでにひどい22％からの減少です。
+
+### ズームと拡大縮小
+
+ユーザーがページやコンテンツを拡大表示できるようにすることが不可欠です。ブラウザの拡大縮小やズーム機能を無効にしようとするテクニックがあります。OSによっては、この有害なパターンを回避するものもありますが、多くはそうでなく回避する必要があるアンチパターンです。
+
+特に弱視の方にはズーム機能が便利です。[世界保健機関](https://www.who.int/news-room/fact-sheets/detail/blindness-and-visual-impairment)によると、「世界では10億人が視力障害を抱えている」とされています。
+
+{{ figure_markup(
+  image="sites-with-zooming-and-scaling-disabled.png",
+  caption="ズームやスケーリングが無効になっているサイト",
+  description="バーチャートを見ると、デスクトップサイトの16.0％、モバイルサイトの19.7％が拡大縮小を無効にしており、デスクトップサイトの21.1％、モバイルサイトの27.1％がズームの上限を1に設定しておりデスクトップサイトの24.4％、モバイルサイトの30.7％がこれらのいずれかを設定していることがわかります。",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vSjkB_XAYiwkYrMuoXp44mdqMGJVDUkTr_48ELozY72Mdv3OlxeWV9ysbDY9bs6hA7LnJTrHar9aZlM/pubchart?oid=1053957382&format=interactive",
+  sheets_gid="1095274901",
+  sql_file="viewport_zoom_scale.sql"
+  )
+}}
+
+デスクトップページの29.34%とモバイルページの30.66%が、`maximum-scale`を1未満の値に設定するか、`user-scalable`を`0`または`none`に設定することで、スケーリングを無効にしようとしていることがわかった。いくつかのOSでは、HTMLで設定されているズームや拡大縮小の無効化に対応していないものがあります。これに対応しているシステムでは、一部のシステムではページが事実上使えなくなることがあります。ブラウザのズームを無効にするのを避ける理由については、Adrian Roselliの記事[Don't Disable Zoom](https://adrianroselli.com/2015/10/dont-disable-zoom.html) を参照してください。
+
+### Language identification
```

Comment:
Missed a translation?

The irony of missing *this* translation - out of all of them 🤣

Reply:
Misstake 😅